packages/ui5-proxy-middleware/package.json

```
@@ -0,0 +1,44 @@
+{
+    "name": "@sap-ux/ui5-proxy-middleware",
+    "description": "UI5 proxy middleware",
+    "version": "0.0.1",
+    "license": "Apache-2.0",
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "clean": "rimraf dist",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --ci --forceExit --detectOpenHandles --colors",
+        "watch": "tsc --watch"
+    },
+    "files": [
+        "LICENSE",
+        "dist",
+        "ui5.yaml"
+    ],
+    "dependencies": {
+        "@sap-ux/logger": "workspace:*",
+        "express": "4.17.2",
+        "http-proxy-middleware": "2.0.1",
+        "https-proxy-agent": "5.0.0",
+        "i18next": "20.3.2",
+        "yaml": "2.0.0-10"
+    },
+    "devDependencies": {
+        "@types/express": "4.17.13",
+        "@ui5/server": "2.4.0",
+        "@ui5/project": "2.6.0",
+        "mock-http-server": "1.4.5",
+        "request": "2.88.2",
+        "request-promise": "4.2.6"
+    },
+    "ui5": {
+        "dependencies": []
+    },
+    "engines": {
+        "pnpm": ">=6.0.2",
```

```suggestion
        "pnpm": ">=6.26.1",
```
Other packages in main have this version.

---

Thanks, will change

---

Done with https://github.com/SAP/open-ux-tools/pull/249/commits/d8139eb45bbd2fe71f4f477a218a5739bdb3dca3